.github/workflows/macOS_gcc10.yml

```diff
@@ -8,7 +8,8 @@ env:
   BUILD_TYPE: RelWithDebInfo
   # Unfortunately the tests for the Xerces plugin fail: https://travis-ci.org/ElektraInitiative/libelektra/jobs/483331657#L3740
   # The curlget tests fail: https://github.com/ElektraInitiative/libelektra/issues/3382
-  PLUGINS: 'ALL;-xerces;-curlget'
+  # The tests fail with jni: https://github.com/ElektraInitiative/libelektra/issues/3747
+  PLUGINS: 'ALL;-xerces;-curlget;-jni'
```

Comment:
@mpranj Can you write an issue why you need to exclude jni here? @tucek works on the improvement of the Java bindings&plugins, maybe he finds a solution.

Reply:
Or is it #3747?

Reply:
> Or is it #3747?

Exactly, it's reported and noted in the lines above the exclusion. When possible I link to the issue in a comment near the exclusion.